src/Neo/Network/P2P/Payloads/Signer.cs

```diff
@@ -66,6 +67,31 @@ public class Signer : IInteroperable, ISerializable
             /*AllowedGroups*/       (Scopes.HasFlag(WitnessScope.CustomGroups) ? AllowedGroups.GetVarSize() : 0) +
             /*Rules*/               (Scopes.HasFlag(WitnessScope.WitnessRules) ? Rules.GetVarSize() : 0);
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public bool Equals(Signer other)
+        {
+            if (ReferenceEquals(this, other))
+                return true;
+            if (other is null) return false;
+            return Account == other.Account &&
+                Scopes == other.Scopes &&
+                AllowedContracts.AsSpan().SequenceEqual(other.AllowedContracts.AsSpan()) &&
```

Comment:
Serialize the object for make the compare seems very weird, and slow, i will fix it tomorrow

Reply:
> Serialize the object for compare seems very weird, and slow, i will fix it tomorrow

What was serialized？

Reply:
> Serialize the object for compare seems very weird, and slow, i will fix it tomorrow

Span is faster